Flat/interface/PandaAnalyzer.h

```diff
@@ -208,6 +217,8 @@ public :
     void CalcBJetSFs(BTagType bt, int flavor, double eta, double pt, 
                      double eff, double uncFactor, double &sf, double &sfUp, double &sfDown);
     void ComplicatedLeptons();
+    void ComplicatedPhotons();
+    bool PFChargedPhotonMatch(const panda::Photon& photon);
```

Comment:
We have many modules, so it is good to alphabetize this

Reply:
Agree. I didn't even notice until now it was following an alphabetical order, I moved down the PFChargedPhotonMatch function